tools/form-generation-tool.adoc

```diff
@@ -14,21 +14,21 @@ Before you begin the installation, please be sure your machine meets the followi
 - Node 18 (To install it, follow these https://nodejs.org/en/download/package-manager/[instructions,window=_blank])
 
 
-Once the prerequisites are met, it is easy to install *{FORM_GENERATION_TOOL}*.
+Once the prerequisites are met, it is easy to install the *{FORM_GENERATION_TOOL}*.
 It is available as the `@kie-tools/form-generation-tool` `npm` https://www.npmjs.com/package/@kie-tools/form-generation-tool[package,window=_blank].
-To installl this package, simply issue the `npm` command below or any other command you use for installing `npm` packages:
+To install this package, simply issue the `npm` command below or any other command you use for installing `npm` packages:
 
 [source,bash]
 ----
 npm install -g @kie-tools/form-generation-tool
 ----
 
 
-NOTE: Please use version higher than *0.32.0*.
+NOTE: Please use version *0.32.0* or later.
```

Comment:
@jomarko, Sorry I forgot to ask, will version 0.32.0 not be a valid version?
i.e. should I change it to Please use a version later than *0.32.0*.

Reply:
@kbowers-ibm thank you for the question , **0.32.0** is not a valid version, so to answer your question, yes, you should use **Please use a version later than 0.32.0.**